sass/main.scss

```
@@ -357,29 +357,62 @@ nav {
         }
     }
 
-    #mlh {
-        @include button;
-        @include button-effect($light-blue);
-        background-color: $light-blue;
+    #hackathon-buttons {
+        display: flex;
+        flex-direction: column;
         margin: 0 auto;
-        padding: 30px;
-        border-radius: 20px;
-
-        i, p {
-            display: inline;
-        }
-
-        p {
-            font-size: 1.2em;
-            font-weight: $font-medium;
+        align-items: flex-start;
+
+        #mlh {
+            @include button;
+            @include button-effect($light-blue);
+            background-color: $light-blue;
+            margin: 24px 0;
```

To be consistent in units:
```suggestion
            margin: 24px 0px;
```